doc/source/whatsnew/v0.20.0.rst

```diff
@@ -1086,50 +1085,50 @@ joins, :meth:`DataFrame.join` and :func:`merge`, and the ``.align`` method.
 
 - ``Index.intersection``
 
-  .. ipython:: python
+.. ipython:: python
 
-     left = pd.Index([2, 1, 0])
-     left
-     right = pd.Index([1, 2, 3])
-     right
+   left = pd.Index([2, 1, 0])
+   left
+   right = pd.Index([1, 2, 3])
+   right
 
-  Previous Behavior:
+Previous Behavior:
 
   .. code-block:: ipython
```

Comment:
Why the indentation? I think the directive needs to start unindented.

Reply:
I should not remove the `  Previous Behavior:` . There is an indentation due to a bullet point. But I had some other blocks I shouldn't index. So remove indents in other blocks. 👍

Reply:
I meant for `  .. code-block:: ipython`. If I'm not mistaken the indentation would create a `\<blockquote>`. If that's wanted, the following code lines have to be indented with at least 3 more spaces than the `.. code-block::`

```
   .. code-block:: python

      code-lines
```

Reply:
Done